### PR TITLE
Refactor FXIOS-7677 [v121] Remove SnapKit from WebsiteDataManagementViewController

### PR DESCRIPTION
**Pull request by ksoni-1258**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7677)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/17114)

## :bulb: Description
- Remove SnapKit from WebsiteDataManagementViewController  #17114

## :pencil: Checklist
You have to check all boxes before merging
- [X] Filled in the above information (tickets numbers and description of your work)
- [X] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Comment by ksoni-1258** [comment]
added image

File: Client/Frontend/Settings/TPAccessoryInfo.swift [hunk 1]
```diff
@@ -49,9 +49,9 @@ class TPAccessoryInfo: ThemedTableViewController {
         topStack.isLayoutMarginsRelativeArrangement = true
 
         sep.backgroundColor = themeManager.currentTheme.colors.borderPrimary
-        sep.snp.makeConstraints { make in
-            make.height.equalTo(0.5)
-            make.width.equalToSuperview()
+        sep.heightAnchor.constraint(equalToConstant: 0.5).isActive = true
+        if let sepSuperView = sep.superview {
+            sep.widthAnchor.constraint(equalTo: sepSuperView.widthAnchor).isActive = true
```
Comment (lmarceau) [inline Client/Frontend/Settings/TPAccessoryInfo.swift:54]: This PR includes changes that were done for the other task in https://github.com/mozilla-mobile/firefox-ios/pull/17134 when it shouldn't. Can you remove those (you can revert the changes to that file, and push again on that same branch).

Comment (ksoni-1258) [reply Client/Frontend/Settings/TPAccessoryInfo.swift]: Made the changes as per the step above, Kindly review the PR @lmarceau

**Review by lmarceau — APPROVED** [review]

**Comment by lmarceau** [comment]
@ksoni-1258 There's a failing UI tests in that area, so seems this needs to be investigated. Could you have a look into it ? 

Screenshot 2023-11-02 at 3 33 23 PM

**Comment by ksoni-1258** [comment]
- Fixed the issue of constraints.
- Another point of failure: In the UI Test case  DataManagementTests ->  testWebSiteDataEnterFirstTime (Line 40) the term "Website Data" needs to replaced by "Web Site Data" as this is seen in UI. May i make the required changes in UI Tests also, or this is due to any other point of failure?

**Comment by lmarceau** [comment]
@ksoni-1258 I believe the UI tests is searching for `.SettingsWebsiteDataTitle` which is the "Website Data" string, hence why we don't need to adjust it (and it's green ✅). Thank you for the work!
